README.md

```diff
@@ -3,55 +3,76 @@
 ## Install
 
 - This assumes an environment is already set up for npm packages - if not, please use npm init following the steps at [https://docs.npmjs.com/getting-started/using-a-package.json](https://docs.npmjs.com/getting-started/using-a-package.json).
-- run `npm install @patternfly/patternfly-next --save`
+- run `npm install @patternfly/patternfly-next`
 
-When you install PatternFly Next, the package includes:
+Any of the files below are meant for use in consuming PatternFly's styles. The recommended consumption approach will vary from project to project.
 
-- a single file for the entire compiled library: `node_modules/@patternfly/patternfly-next/patternfly.css`
-- individual files with each component compiled separately: `node_modules/@patternfly/patternfly-next/<ComponentName>/styles.css`
-- a single file for the entire library's source (SASS): `node_modules/@patternfly/patternfly-next/patternfly.scss`
-- individual files for each component's source (SASS): `node_modules/@patternfly/patternfly-next/<ComponentName>/styles.scss`
+- a single file for the complete PatternFly styles:
 
-Any of the files above are meant for use in consuming the library. The recommended
-consumption approach will vary from project to project.
+  ```
+  node_modules/@patternfly/patternfly-next/patternfly.css
+  ```
+
+- individual stylesheets for each of PatternFly's blueprints (e.g. components, layout):
```

Comment:
We should discuss - we aren't using blueprints as a naming convention for consumers - please use the original language here.

Reply:
It's great that these instructions document the existence of component stylesheets and help users who need a hand finding them. But the original language doesn't document the existence of demo, layout, and utility stylesheets, or help those same users find them. When I saw how much more the dependency made easily available, I assumed the README had been written at point when there _was_ nothing beyond components. Definitely true that "blueprint" is an unfamiliar term — it's only in patternfly-cli. How about just being explicit, like

> - Compiled CSS stylesheets for each of PatternFly's components and layouts, located at
>    ```
>    node_modules/@patternfly/patternfly-next/components/<ComponentName>/<component-name>.css
>    ```
>    and
>     ```
>    node_modules/@patternfly/patternfly-next/layout/<LayoutName>/<layout-name>.css
>    ```
> - SASS source files for each of PatternFly's components and layouts, located at
>    ```
>    node_modules/@patternfly/patternfly-next/components/<ComponentName>/<component-name>.scss
>     ```
>    and
>     ```
>    node_modules/@patternfly/patternfly-next/layout/<LayoutName>/<layout-name>.scss
>    ```

Alternatively you could list the full `patternfly.scss`, `patternfly.css`, and `patternfly.min.css`, and add that "CSS and SASS files for individual components and layouts, along with other base styles and utilities, can be found in `node_modules/@patternfly/patternfly-next`"